### ISSUE TITLE
Reserve a test set geographically

### ISSUE THREAD

**Issue by metazool** [MAMBO-Horizon-WP4/att-unet-shrub-id#23]
Rather than reserve a test dataset when we generate training data (because there will be a likelihood of patch overlaps) we should reserve one before (a region that we just remove from the raster and filter out of the vectors).

Rafa's original data did have this (a single large TIFF with associated polygons) - we changed the raster layer but it's worth checking the spatial extent of that test data and replicating it, if we've got annotations
